### ISSUE TITLE
==> default: Error: Evaluation Error: Error while evaluating a Resource Statement

### ISSUE THREAD

**Issue by SonkoDmitry** [puphpet/puphpet#2053]
After vagrant up I have tis error
```
==> default: Error: Evaluation Error: Error while evaluating a Resource Statement, Class[Puphpet::Adminer]: has no parameter named 'php_package' at /tmp/vagrant-puppet/manifests-75f35e3bc7e32744860c4bb229c88812/Postgresql.pp:142:5 on node takeguru
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```
full log:
```
$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'puphpet/ubuntu1404-x64'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'puphpet/ubuntu1404-x64' is up to date...
==> default: Setting the name of the VM: takeguru_default_1450374926031_67789
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: Forwarding ports...
    default: 22 => 6029 (adapter 1)
    default: 80 => 8080 (adapter 1)
    default: 22 => 2222 (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: Warning: Connection timeout. Retrying...
==> default: Machine booted and ready!
GuestAdditions seems to be installed (5.0.10) correctly, but not running.
Usage: service < option > | --status-all | [ service_name [ command | --full-restart ] ]
bash: line 3: vboxadd: command not found
Removing existing VirtualBox DKMS kernel modules ...done.
Removing existing VirtualBox non-DKMS kernel modules ...done.
Building the VirtualBox Guest Additions kernel modules ...done.
Doing non-kernel setup of the Guest Additions ...done.
You should restart your guest to make sure the new modules are actually used

Restarting VM to apply changes...
==> default: Attempting graceful shutdown of VM...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: Warning: Connection timeout. Retrying...
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
==> default: Setting hostname...
==> default: Configuring and enabling network interfaces...
==> default: Exporting NFS shared folders...
==> default: Preparing to edit /etc/exports. Administrator privileges will be required...
==> default: Mounting NFS shared folders...
==> default: Mounting shared folders...
    default: /vagrant => /Users/dmitry/work/takeguru/takeguru
    default: /tmp/vagrant-puppet/modules-b4889f66312f5704883c9d39f581ab2f => /Users/dmitry/work/takeguru/takeguru/puphpet/puppet/modules
    default: /tmp/vagrant-puppet/manifests-75f35e3bc7e32744860c4bb229c88812 => /Users/dmitry/work/takeguru/takeguru/puphpet/puppet/manifests
==> default: Updating /etc/hosts file on active guest machines...
==> default: Updating /etc/hosts file on host machine (password may be required)...
==> default: Running provisioner: hostmanager...
==> default: Running provisioner: shell...
    default: Running: inline script
==> default: Running provisioner: shell...
    default: Running: /var/folders/95/9z_9lg4j41nbl6jr959sltvc0000gn/T/vagrant-shell20151217-873-1ywwt9l.sh
==> default: 
==> default:  ____        ____  _   _ ____      _      generated using
==> default: |  _ \ _   _|  _ \| | | |  _ \ ___| |_   ___ ___  _ __ ___
==> default: | |_) | | | | |_) | |_| | |_) / _ \ __| / __/ _ \| '_ ` _ \
==> default: |  __/| |_| |  __/|  _  |  __/  __/ |_ | (_| (_) | | | | | |
==> default: |_|    \__,_|_|   |_| |_|_|   \___|\__(_)___\___/|_| |_| |_|
==> default: Get:1 http://security.ubuntu.com trusty-security InRelease [64.4 kB]
==> default: Ign http://us.archive.ubuntu.com trusty InRelease
==> default: Get:2 http://us.archive.ubuntu.com trusty-updates InRelease [64.4 kB]
==> default: Get:3 http://security.ubuntu.com trusty-security/main Sources [101 kB]
==> default: Get:4 http://security.ubuntu.com trusty-security/restricted Sources [4,035 B]
==> default: Get:5 http://security.ubuntu.com trusty-security/universe Sources [31.9 kB]
==> default: Get:6 http://security.ubuntu.com trusty-security/multiverse Sources [2,353 B]
==> default: Ign http://apt.puppetlabs.com trusty InRelease
==> default: Get:7 http://security.ubuntu.com trusty-security/main amd64 Packages [389 kB]
==> default: Hit http://us.archive.ubuntu.com trusty-backports InRelease
==> default: Get:8 http://apt.puppetlabs.com trusty Release.gpg [836 B]
==> default: Hit http://us.archive.ubuntu.com trusty Release.gpg
==> default: Get:9 http://apt.puppetlabs.com trusty Release [32.3 kB]
==> default: Get:10 http://us.archive.ubuntu.com trusty-updates/main Sources [246 kB]
==> default: Get:11 http://security.ubuntu.com trusty-security/restricted amd64 Packages [13.0 kB]
==> default: Get:12 http://apt.puppetlabs.com trusty/PC1 Sources [5,819 B]
==> default: Get:13 http://security.ubuntu.com trusty-security/universe amd64 Packages [122 kB]
==> default: Get:14 http://apt.puppetlabs.com trusty/PC1 amd64 Packages [6,700 B]
==> default: Get:15 http://us.archive.ubuntu.com trusty-updates/restricted Sources [5,359 B]
==> default: Get:16 http://apt.puppetlabs.com trusty/PC1 i386 Packages [6,706 B]
==> default: Get:17 http://us.archive.ubuntu.com trusty-updates/universe Sources [145 kB]
==> default: Get:18 http://security.ubuntu.com trusty-security/multiverse amd64 Packages [4,795 B]
==> default: Get:19 http://security.ubuntu.com trusty-security/main i386 Packages [367 kB]
==> default: Get:20 http://us.archive.ubuntu.com trusty-updates/multiverse Sources [5,167 B]
==> default: Get:21 http://us.archive.ubuntu.com trusty-updates/main amd64 Packages [670 kB]
==> default: Get:22 http://security.ubuntu.com trusty-security/restricted i386 Packages [12.7 kB]
==> default: Get:23 http://security.ubuntu.com trusty-security/universe i386 Packages [122 kB]
==> default: Get:24 http://security.ubuntu.com trusty-security/multiverse i386 Packages [4,972 B]
==> default: Get:25 http://security.ubuntu.com trusty-security/main Translation-en [212 kB]
==> default: Get:26 http://us.archive.ubuntu.com trusty-updates/restricted amd64 Packages [15.9 kB]
==> default: Get:27 http://us.archive.ubuntu.com trusty-updates/universe amd64 Packages [333 kB]
==> default: Get:28 http://security.ubuntu.com trusty-security/multiverse Translation-en [2,437 B]
==> default: Get:29 http://security.ubuntu.com trusty-security/restricted Translation-en [3,206 B]
==> default: Ign http://apt.puppetlabs.com trusty/PC1 Translation-en_US
==> default: Get:30 http://security.ubuntu.com trusty-security/universe Translation-en [71.1 kB]
==> default: Get:31 http://us.archive.ubuntu.com trusty-updates/multiverse amd64 Packages [13.0 kB]
==> default: Ign http://apt.puppetlabs.com trusty/PC1 Translation-en
==> default: Get:32 http://us.archive.ubuntu.com trusty-updates/main i386 Packages [648 kB]
==> default: Get:33 http://us.archive.ubuntu.com trusty-updates/restricted i386 Packages [15.6 kB]
==> default: Get:34 http://us.archive.ubuntu.com trusty-updates/universe i386 Packages [334 kB]
==> default: Get:35 http://us.archive.ubuntu.com trusty-updates/multiverse i386 Packages [13.2 kB]
==> default: Get:36 http://us.archive.ubuntu.com trusty-updates/main Translation-en [334 kB]
==> default: Get:37 http://us.archive.ubuntu.com trusty-updates/multiverse Translation-en [6,832 B]
==> default: Get:38 http://us.archive.ubuntu.com trusty-updates/restricted Translation-en [3,699 B]
==> default: Get:39 http://us.archive.ubuntu.com trusty-updates/universe Translation-en [174 kB]
==> default: Hit http://us.archive.ubuntu.com trusty-backports/main Sources
==> default: Hit http://us.archive.ubuntu.com trusty-backports/restricted Sources
==> default: Hit http://us.archive.ubuntu.com trusty-backports/universe Sources
==> default: Hit http://us.archive.ubuntu.com trusty-backports/multiverse Sources
==> default: Hit http://us.archive.ubuntu.com trusty-backports/main amd64 Packages
==> default: Hit http://us.archive.ubuntu.com trusty-backports/restricted amd64 Packages
==> default: Hit http://us.archive.ubuntu.com trusty-backports/universe amd64 Packages
==> default: Hit http://us.archive.ubuntu.com trusty-backports/multiverse amd64 Packages
==> default: Hit http://us.archive.ubuntu.com trusty-backports/main i386 Packages
==> default: Hit http://us.archive.ubuntu.com trusty-backports/restricted i386 Packages
==> default: Hit http://us.archive.ubuntu.com trusty-backports/universe i386 Packages
==> default: Hit http://us.archive.ubuntu.com trusty-backports/multiverse i386 Packages
==> default: Hit http://us.archive.ubuntu.com trusty-backports/main Translation-en
==> default: Hit http://us.archive.ubuntu.com trusty-backports/multiverse Translation-en
==> default: Hit http://us.archive.ubuntu.com trusty-backports/restricted Translation-en
==> default: Hit http://us.archive.ubuntu.com trusty-backports/universe Translation-en
==> default: Hit http://us.archive.ubuntu.com trusty Release
==> default: Hit http://us.archive.ubuntu.com trusty/main Sources
==> default: Hit http://us.archive.ubuntu.com trusty/restricted Sources
==> default: Hit http://us.archive.ubuntu.com trusty/universe Sources
==> default: Hit http://us.archive.ubuntu.com trusty/multiverse Sources
==> default: Hit http://us.archive.ubuntu.com trusty/main amd64 Packages
==> default: Hit http://us.archive.ubuntu.com trusty/restricted amd64 Packages
==> default: Hit http://us.archive.ubuntu.com trusty/universe amd64 Packages
==> default: Hit http://us.archive.ubuntu.com trusty/multiverse amd64 Packages
==> default: Hit http://us.archive.ubuntu.com trusty/main i386 Packages
==> default: Hit http://us.archive.ubuntu.com trusty/restricted i386 Packages
==> default: Hit http://us.archive.ubuntu.com trusty/universe i386 Packages
==> default: Hit http://us.archive.ubuntu.com trusty/multiverse i386 Packages
==> default: Hit http://us.archive.ubuntu.com trusty/main Translation-en
==> default: Hit http://us.archive.ubuntu.com trusty/multiverse Translation-en
==> default: Hit http://us.archive.ubuntu.com trusty/restricted Translation-en
==> default: Hit http://us.archive.ubuntu.com trusty/universe Translation-en
==> default: Ign http://us.archive.ubuntu.com trusty/main Translation-en_US
==> default: Ign http://us.archive.ubuntu.com trusty/multiverse Translation-en_US
==> default: Ign http://us.archive.ubuntu.com trusty/restricted Translation-en_US
==> default: Ign http://us.archive.ubuntu.com trusty/universe Translation-en_US
==> default: Fetched 4,607 kB in 16s (276 kB/s)
==> default: Reading package lists...
==> default: Running provisioner: shell...
    default: Running: /var/folders/95/9z_9lg4j41nbl6jr959sltvc0000gn/T/vagrant-shell20151217-873-1ab3e97.sh
==> default: Generating public/private rsa key pair.
==> default: Your identification has been saved in /vagrant/puphpet/files/dot/ssh/root_id_rsa.
==> default: Your public key has been saved in /vagrant/puphpet/files/dot/ssh/root_id_rsa.pub.
==> default: The key fingerprint is:
==> default: f0:7c:86:64:08:7d:7b:d3:9e:3c:4b:d2:45:d3:63:3f root@takeguru
==> default: The key's randomart image is:
==> default: +--[ RSA 2048]----+
==> default: |    ..         . |
==> default: |     ....     oo.|
==> default: |      o.o. . ...o|
==> default: |       *..o . .E.|
==> default: |        S.o= o  .|
==> default: |         o. B    |
==> default: |           o o   |
==> default: |            .    |
==> default: |                 |
==> default: +-----------------+
==> default: dpkg-preconfigure: unable to re-open stdin: No such file or directory
==> default: Your private key for SSH-based authentication has been saved to 'puphpet/files/dot/ssh/root_id_rsa'!
==> default: Generating public/private rsa key pair.
==> default: Your identification has been saved in /vagrant/puphpet/files/dot/ssh/id_rsa.
==> default: Your public key has been saved in /vagrant/puphpet/files/dot/ssh/id_rsa.pub.
==> default: The key fingerprint is:
==> default: 6e:7b:ab:60:87:d7:9d:9b:5e:1d:c7:f6:a7:6b:5b:b4 root@takeguru
==> default: The key's randomart image is:
==> default: +--[ RSA 2048]----+
==> default: |                 |
==> default: |                 |
==> default: |                 |
==> default: |               . |
==> default: |        S      .=|
==> default: |       o . . . +=|
==> default: |      + = . o .E=|
==> default: |     . = ..  +.o.|
==> default: |        oo.o+.+o |
==> default: +-----------------+
==> default: Your private key for SSH-based authentication has been saved to 'puphpet/files/dot/ssh/id_rsa'!
==> default: Adding generated root key to /root/.ssh/id_rsa
==> default: Adding generated root key to /root/.ssh/id_rsa.pub
==> default: Adding generated root key to /root/.ssh/authorized_keys
==> default: Adding generated key to /home/vagrant/.ssh/id_rsa
==> default: Adding generated key to /home/vagrant/.ssh/id_rsa.pub
==> default: Adding generated key to /home/vagrant/.ssh/authorized_keys
==> default: Running provisioner: shell...
    default: Running: /var/folders/95/9z_9lg4j41nbl6jr959sltvc0000gn/T/vagrant-shell20151217-873-1trnrzm.sh
==> default: Running provisioner: shell...
    default: Running: /var/folders/95/9z_9lg4j41nbl6jr959sltvc0000gn/T/vagrant-shell20151217-873-1hzvvup.sh
==> default: Successfully installed deep_merge-1.0.1
==> default: Successfully installed activesupport-4.2.5
==> default: Successfully installed vine-0.2
==> default: 3 gems installed
==> default: Running provisioner: puppet...
==> default: Running Puppet with ...
==> default: Info: Loading facts
==> default: Info: Loading facts
==> default: Info: Loading facts
==> default: Info: Loading facts
==> default: Info: Loading facts
==> default: Info: Loading facts
==> default: Info: Loading facts
==> default: Info: Loading facts
==> default: Info: Loading facts
==> default: Info: Loading facts
==> default: Info: Loading facts
==> default: Info: Loading facts
==> default: Info: Loading facts
==> default: Error: Evaluation Error: Error while evaluating a Resource Statement, Class[Puphpet::Adminer]: has no parameter named 'php_package' at /tmp/vagrant-puppet/manifests-75f35e3bc7e32744860c4bb229c88812/Postgresql.pp:142:5 on node takeguru
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

**Comment by SonkoDmitry** [comment]
Holly ..., i need two db servers on vagrant - maria and postgres. In config.yml, in maria section I lost ckeck adminer, but in postgres adminer checked. After change adminer: 0 to adminer: '1' in mariadb section all start successful

**Comment by SonkoDmitry** [comment]
But one more trouble, in the end of vagrant up after change config, I take this:
```
...
==> default: Notice: /Stage[main]/Puphpet_rabbitmq/Puphpet::Php::Pecl[amqp]/Php::Pecl::Module[amqp]/Exec[pecl-amqp]/returns: No information found for `amqp'
==> default: Error: printf "\n" | pecl -d preferred_state=stable install amqp && pecl info amqp returned 1 instead of one of [0]
==> default: Error: /Stage[main]/Puphpet_rabbitmq/Puphpet::Php::Pecl[amqp]/Php::Pecl::Module[amqp]/Exec[pecl-amqp]/returns: change from notrun to 0 failed: printf "\n" | pecl -d preferred_state=stable install amqp && pecl info amqp returned 1 instead of one of [0]
==> default: Notice: /Stage[main]/Puphpet_php/Service[php5-fpm]: Dependency Exec[pecl-amqp] has failures: true
==> default: Warning: /Stage[main]/Puphpet_php/Service[php5-fpm]: Skipping because of failed dependencies
==> default: Info: /Stage[main]/Puphpet_php/Service[php5-fpm]: Unscheduling all events on Service[php5-fpm]
==> default: Info: Class[Puphpet_php]: Unscheduling all events on Class[Puphpet_php]
==> default: Info: Class[Puphpet_rabbitmq]: Unscheduling all events on Class[Puphpet_rabbitmq]
==> default: Info: Stage[main]: Unscheduling all events on Stage[main]
==> default: Info: Creating state file /opt/puppetlabs/puppet/cache/state/state.yaml
==> default: Notice: Applied catalog in 779.48 seconds
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

**Comment by jtreminio** [comment]
Fixed by 806a2ee7d8956be5ad31ee317dd01061c689c939
